xbmc/cores/RetroPlayer/savestates/SavestateDatabase.cpp

```diff
@@ -146,16 +146,18 @@ bool CSavestateDatabase::GetSavestatesNav(CFileItemList& items,
 
     const std::string label = savestate->Label();
 
+    CDateTime date = savestate->Created() - CDateTime::GetTimezoneBias();
```

Comment:
No math spread through code, use a CDateTime function that gives you what you want